src/map/ai/ai_char_normal.cpp

```diff
@@ -2531,22 +2531,15 @@ void CAICharNormal::ActionWeaponSkillFinish()
     if (m_PChar->GetMJob() == JOB_THF && m_PChar->StatusEffectContainer->HasStatusEffect(EFFECT_TRICK_ATTACK))
     {
         taChar = battleutils::getAvailableTrickAttackChar(m_PChar, m_PBattleTarget);
-        if (taChar != NULL) damage += m_PChar->AGI();
+        if (taChar != NULL && damage >= 0) damage += m_PChar->AGI();
```

Comment:
er.. this isn't even close to right (not your change, but what was there before...)

lets just get rid of it.. (it should be in the script, because it has to get multiplited by ftp - maybe it needs a binding)